model/src/main/kotlin/io/spine/validation/PatternPolicy.kt

```diff
@@ -93,12 +92,12 @@ private fun FieldType.isSupported(): Boolean = isSingularString || isRepeatedStr
  * The property is `public` because the option generator also uses it.
  */
 public val FieldType.isRepeatedString: Boolean
-    get() = isList && list.primitive == TYPE_STRING
```

Comment:
We don't have to check it because otherwise `list.primitive` would have a default value, which is not `TYPE_STRING`. The same is above.